chart-diagrams/Graphics/Rendering/Chart/Backend/Diagrams.hs

```diff
@@ -568,7 +568,7 @@ fontStyleToTextOpts env =
       , F.textHeight = scaledH -- _font_size fs
       }
 
-fontFromName :: (Read n, RealFloat n) => String -> F.PreparedFont n
+fontFromName :: (Read n, RealFloat n) => String -> IO (F.PreparedFont n)
```

Comment:
Note: this is a breaking change.

Reply:
This `fontFromName` function is neither exported nor used anywhere inside the module. I think it can be safely removed. Did I miss something?